### PR TITLE
RIseBidAdapter: support currency

### PR DESCRIPTION
**Pull request by YakirLavi**
## Type of change
- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter 
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Support Currency in adapter bid request.

### CODE REVIEW

File: modules/riseBidAdapter.js [hunk 1]
```diff
@@ -147,11 +147,11 @@ function getFloor(bid, mediaType) {
     return 0;
   }
   let floorResult = bid.getFloor({
-    currency: CURRENCY,
+    currency: bid.params.currency || DEFAULT_CURRENCY,
```
Comment (noamtzu) [inline modules/riseBidAdapter.js:150]: 1. please select from 3 options
2. extract the currency selection to a function or a var

File: test/spec/modules/riseBidAdapter_spec.js [hunk 2]
```diff
@@ -195,6 +195,12 @@ describe('riseAdapter', function () {
       expect(request.data.bids[1].mediaType).to.equal(BANNER)
     });
 
+    it('should send the correct currency in bid request', function () {
+      const expectedCurrency = 'USD';
+      const request = spec.buildRequests(bidRequests, bidderRequest)
+      expect(request.data.bids[0].currency).to.equal(expectedCurrency);
+    });
+
     it('should respect syncEnabled option', function() {
```
Comment (noamtzu) [inline test/spec/modules/riseBidAdapter_spec.js:204]: add a test of EUR currency as a bidder param

**Review by noamtzu — CHANGES_REQUESTED** [review]
1. Edit the md file in this repo
2. The same comments also relevant for MM

File: modules/riseBidAdapter.js (hunk 1 above)
Comment (YakirLavi) [reply modules/riseBidAdapter.js]: Done

File: test/spec/modules/riseBidAdapter_spec.js (hunk 2 above)
Comment (YakirLavi) [reply test/spec/modules/riseBidAdapter_spec.js:218]: Done

**Pull request merged**